docs/2.8.0/versions.json

```diff
@@ -1,5 +1,5 @@
 {
-  "daml": "2.7.0-snapshot.20230719.11983.0.vd999a21a",
+  "daml": "cbcaca974ac24c69246c08f167f20b759ebfade6",
```

Comment:
I messed up, https://github.com/digital-asset/daml/pull/17390 isn't part of today's release. do I have to wait for the next release or is it okay to point to (the current commit of) `main`?